RELEASE_INFO-6.7.md

```diff
@@ -27,6 +27,10 @@ The following classes and constants were deprecated as they will not be used any
 Additionally, the following configuration was deprecated:
 * `shopware.cache.invalidation.http_cache`
 
+### Added ResetInterface to LineItemGroupBuilder
```

Comment:
we do not need to document bugfixes in the release information. only relevant stuff for other developers